examples/dgmulti_3d/elixir_euler_curved.jl

```diff
@@ -1,7 +1,7 @@
 
 using Trixi, OrdinaryDiffEq
 
-dg = DGMulti(polydeg = 3, element_type = Hex(), approximation_type = SBP(),
+dg = DGMulti(polydeg = 3, element_type = Hex(), approximation_type = GaussSBP(),
```

Comment:
Why? This looks like a totally unrelated change, isn't it?

Reply:
Yeah, that slipped though when updating the test values.